package/Editor/PromptWindow/PromptWindowUI.cs

```diff
@@ -137,13 +137,15 @@ public void Render(Rect position)
             bool shouldAutoGenerateSeed = imagesliderValue > 1;
             if (shouldAutoGenerateSeed) { seedinputText = "-1"; }
 
-            CustomStyle.ButtonPrimary("Generate Image", 40, () =>
+            CustomStyle.ButtonPrimary("Generate Image", 40, 0, () =>
             {
                 promptinputText = SerializeTags(tags);
                 negativepromptinputText = SerializeTags(negativeTags);
 
                 EditorPrefs.SetString("postedModelName", DataCache.instance.SelectedModelId);
 
+                PromptFetcher.IsWorkflow = false;
```

Comment:
What is this used for ?

Reply:
can you change the name accordingly ? Something like "silentFetch" or "silenceMode" or something in this flavor ?